distribution/lib/Standard/Base/0.0.0-dev/src/System/File.enso

```diff
@@ -8,7 +8,8 @@ import Standard.Base.Data.Text.Matching_Mode
 import Standard.Base.Data.Text.Text_Sub_Range
 from Standard.Base.Data.Text.Encoding as Encoding_Module import Encoding, Encoding_Error
 from Standard.Base.Error.Problem_Behavior as Problem_Behavior_Module import Problem_Behavior, Report_Warning
-from Standard.Base.Runtime.Resource import all
+import Standard.Base.Runtime.Resource
+from Standard.Base.Runtime.Resource import Managed_Resource
```

Comment:
```suggestion
from Standard.Base.Runtime.Resource import Managed_Resource
```

I'm not sure if this is the desired behaviour, but IIRC currently lines like
```
from Standard.Base.Runtime.Resource import Managed_Resource
```
already imply also
```
import Standard.Base.Runtime.Resource
```

i.e. just `from Standard.Base.Runtime.Resource import Managed_Resource` brings the `Resource` module into scope as well, so the other line is unnecessary.

Unless that behaviour is a bug?

Reply:
Have raised this as a question on the modules work as this feels incorrect.